From bf19ba514fa7b97763aff603b6a078c7bcc86b21 Mon Sep 17 00:00:00 2001
From: Thomas Moschny <thomas.moschny@gmx.de>
Date: Wed, 30 May 2012 12:54:39 +0200
Subject: [PATCH] Desktop file fixes.

- Remove an extra space after the group header.
- Add required key 'Type'.
---
 awesome.desktop |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/awesome.desktop b/awesome.desktop
index b88e1d5..8a7f354 100644
--- a/awesome.desktop
+++ b/awesome.desktop
@@ -1,6 +1,7 @@
-[Desktop Entry] 
+[Desktop Entry]
 Encoding=UTF-8
 Name=awesome
 Comment=Highly configurable framework window manager
 TryExec=awesome
 Exec=awesome
+Type=Application
-- 
1.7.7.6

